packages/xo-server/src/api/vtpm.mjs

```diff
@@ -0,0 +1,27 @@
+export function create({ vm }) {
+  return this.getXapi(vm).VTPM_create({ VM: vm._xapiRef })
```

Comment:
What should this method return?

Reply:
It returns the VTPM opaque ref. Should it return the VTPM UUID instead?

Reply:
What's the intended use for this API?

Reply:
I'm not sure to understand the question but,  as `xo-server` generally uses `UUID`s to manage objects, it seems consistent to return the `UUID` of the VTPM when creating it.